packages/manager/modules/pci/src/projects/project/storages/object-storage/users/users.controller.js

```diff
@@ -87,4 +95,11 @@ export default class PciStoragesContainersUsersController {
       ),
     );
   }
+
+  trackPage(action) {
```

Comment:
**suggestion:**  Since the parameter is `label` and not an action. 
```suggestion
  trackPage(label) {
```

Reply:
Okay Anoop